devicetypes/smartthings/zwave-accessory-switch-generic.src/zwave-accessory-switch-generic.groovy

```diff
@@ -0,0 +1,177 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Z-Wave Accessory Switch Generic", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true) {
```

Comment:
@mckeed I don't think this can execute locally because of the conditional code in `off()`. Thoughts?

Reply:
Yes, now that I can actually test local execution, I can see that all MSR data value checks seem to return false. Local execution will probably require either different approach or configuring dimmer's ramp time to 0 during installation.  Or we can transfer execution to cloud.